File: src/main/java/com/kingdom/model/GameLog.java
package com.kingdom.model;


import javax.persistence.*;

@Table(name = "game_log")
@Entity
public class GameLog {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "logid")
    private int logId;

    @Column(name = "gameid")
    private int gameId;

    private String log;

    public int getLogId() {
        return logId;
    }

    public void setLogId(int logId) {
        this.logId = logId;
    }

    public int getGameId() {
        return gameId;
    }

    public void setGameId(int gameId) {
        this.gameId = gameId;
    }

    public String getLog() {
        return log;
    }

    public void setLog(String log) {
        this.log = log;
    }
}

File: src/main/kotlin/com/kingdom/model/GameLog.kt
package com.kingdom.model


import javax.persistence.*

@Table(name = "game_log")
@Entity
class GameLog {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "logid")
    var logId: Int = 0

    @Column(name = "gameid")
    lateinit var gameId: String

    var log: String? = null
}
